test/dash-playlist-loader.test.js

```diff
@@ -471,7 +472,7 @@ QUnit.test('requestSidx_: creates an XHR request for a sidx range', function(ass
   const sidxInfo = {
     resolvedUri: 'sidx.mp4',
     byterange: {
-      offset: 0,
+      offset: 10,
```

Comment:
We don't want to re-use bytes in this test (from the container request) so I set the offset so that it doesn't happen.